test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/RestoreCommandTests.cs

```diff
@@ -1168,6 +1168,211 @@ public async Task Restore_PackageNamespace_Fails()
             }
         }
 
+        [Fact]
+        public async Task Restore_PackageNamespace_PassAllNecessarySourceOption_Succeed()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var net461 = NuGetFramework.Parse("net461");
+
+                var projectA = SimpleTestProjectContext.CreateLegacyPackageReference(
```

Comment:
You're creating a LegacyPackageReference and a packages.config next to it, which really isn't supported.

I think there's a bug in this `CreateLegacyPackageReference` where it doesn't *always* create PackageReference projects.

Reply:
Ok. Since `-source` [not supported](https://github.com/NuGet/NuGet.Client/pull/4214/files#r694429014) for PR type I decided to remove new tests in this file. We already have enough PC type tests.